src/timeseriesflattener/flattened_dataset.py

```diff
@@ -1,22 +1,28 @@
-from typing import Callable, Dict, List, Optional, Tuple, Union
+from multiprocessing import Pool
+from typing import Callable, Dict, List, Optional, Union
 
 import catalogue
 import pandas as pd
 from pandas import DataFrame
+from wasabi import msg
 
 resolve_fns = catalogue.create("timeseriesflattener", "resolve_strategies")
 
 from timeseriesflattener.resolve_multiple_functions import *
```

Comment:
Exactly a good reason to not use wildcard imports ;) - There is a point on the [FAQ](https://pypi.org/project/catalogue/) in the catalogue package. They suggest entry points. Another easy fix is simply importing the function you added the decorator to (you can add the comment #noqa to remove flake8 or similar linter issue for unused function import)

Reply:
Haha, I'd say this is a case where wildcard imports make sense because we only have resolve-strategies in resolve_multiple_functions ;-)

I'll just import 'em all 👍

Reply:
> Haha, I'd say this is a case where wildcard imports make sense because we only have resolve-strategies in resolve_multiple_functions ;-)

Not sure many would agree, but maybe we are just snobby programmers 🎩

Reply:
Haha, nah, it's a robustness vs. speed trade-off, I totally see the argument 👍